openstack/tools/upload_octavia_amphora_image.sh

```diff
@@ -1,2 +1,14 @@
 #!/bin/bash -eu
-juju run octavia-diskimage-retrofit/0 retrofit-image
+
+source $(realpath $(dirname $0))/../novarc
+
+IMAGE_ID=$(openstack image list --name jammy --format value --column ID)
```

Comment:
So this script previously took a release name via '--release foo' which is useful when running on a focal deploy. Defaulting to jammy won't work there so perhaps it should be put back, unless we can determine the release from something else?

Reply:
Ok, that's a good point. I'll work on this PR a bit more.

Reply:
I believe this is not necessary if we have https://github.com/canonical/stsstack-bundles/pull/217 i.e. of an image exists in glance that corresponds to the version/release deployed (focal->ussuri, jammy->yoga etc).